sys/targets/targets.go

```diff
@@ -775,6 +778,60 @@ func initTarget(target *Target, OS, arch string) {
 		target.ExecutorUsesForkServer = false
 		target.HostFuzzer = true
 	}
+	target.initAddr2Line()
+}
+
+func (target *Target) initAddr2Line() {
+	// Initialize addr2line lazily since lots of tests don't need it,
+	// but we invoke a number of external binaries during addr2line detection.
+	var (
+		init sync.Once
+		bin  string
+		err  error
+	)
+	target.Addr2Line = func() (string, error) {
+		init.Do(func() { bin, err = target.findAddr2Line() })
+		return bin, err
+	}
+}
+
+func (target *Target) findAddr2Line() (string, error) {
+	// First try to find llvm-addr2line as it's significantly faster on large binaries.
+	if path, err := exec.LookPath("llvm-addr2line"); err == nil {
+		return path, nil
+	}
+	// It may also be installed only as a versioned binary,
```

Comment:
Do we really need this logic in the code?
I think we'd better add one more `update-alternatives` to our Dockerfile.

Reply:
+1 to fix our docker

Reply:
Done